tests/cypress/tests-e2e/print.cy.js

```diff
@@ -0,0 +1,803 @@
+                                        },
+                                        geodetic: {
+                                            type: 'boolean',
+                                            default: false,
+                                        },
+                                        fontSize: {
+                                            type: 'int',
+                                            default: 8,
+                                        },
+                                        fontColor: {
+                                            type: 'String',
+                                            default: 'black',
+                                        },
+                                        font: {
+                                            type: 'String',
+                                            default: 'Helvetica',
+                                        },
+                                        barBgColor: {
+                                            type: 'String',
+                                            default: 'white',
+                                        },
+                                        labelRotation: {
+                                            type: 'float',
+                                            default: 0,
+                                        },
+                                    },
+                                },
+                                {
+                                    name: 'northArrow',
+                                    type: 'NorthArrowAttributeValues',
+                                    clientParams: {
+                                        graphic: {
+                                            type: 'String',
+                                            default: 'file://ressources/north_arrow.png',
+                                        },
+                                        backgroundColor: {
+                                            type: 'String',
+                                            default: 'rgba(255, 255, 255, 0)',
+                                        },
+                                    },
+                                },
+                            ],
+                        },
+                    ],
+                    smtp: {
+                        enabled: false,
+                    },
+                    formats: ['bmp', 'gif', 'jpeg', 'jpg', 'pdf', 'png', 'svg', 'tif', 'tiff'],
+                },
+            })
+        }).as('capabilities')
+    }
+
+    function interceptPrintRequest() {
+        cy.intercept('POST', '**/report.pdf', (req) => {
+            req.reply({
+                body: {
+                    ref: printID,
+                    statusURL: '/print/status/' + printID,
+                    downloadURL: '/print/report/' + printID,
+                },
+                delay: 200,
+            })
```

Comment:
Small response mockup like here are fine inline as the opposite of the capabilities which should be in a fixture file